test-variables.yml.example

```diff
@@ -4,6 +4,7 @@ BROWSER:
   # --disable-dev-shm-usage  and --no-sandbox are required for running chromedriver in a container
   OPTIONS: add_argument("--ignore-certificate-errors");add_argument("window-size=1920,1024");add_argument("--disable-dev-shm-usage");add_argument("--no-sandbox")
 OCP_CONSOLE_URL: "http://console-openshift-console.apps.my-cluster.test.redhat.com/"
+OCP_API_URL: "https://api.mycluster.redhat.com:6443"
```

Comment:
Can we derive this from OCP_CONSOLE_URL, so that we can avoid one more extra variable.

Reply:
There seems to be a pattern between the 2 URLs, so probably. However, this pattern could change one day. 

May be a better design would be: 
* only ask for the API URL
* do the `oc login` against it 
* determine the console's URL from that.

Reply:
it could be the other way around, from the API URL you can query anything, including the console URL